src/index.d.ts

```diff
@@ -469,6 +469,12 @@ declare namespace admin.messaging {
     launchImage?: string;
   };
 
+  type CriticalSound = {
+    critical?: boolean;
```

Comment:
Isn't it redundant since it's of type CriticalSound and it should be always true?

Reply:
Technically, following is allowed in APNs:

```
"sound": {
  "name": "default"
}
```

This is same as:

```
"sound": "default"
```

Reply:
It can be false. We are using the APNs terminology here. So this is fine. Plus the API change has already been approved.